include/admin/comments.inc.php

```diff
@@ -375,7 +375,7 @@
 $data['get']['filter'] = $serendipity['GET']['filter']; // don't trust {$smarty.get.vars} if not proofed, as we often change GET vars via serendipty['GET'] by runtime
 $data['commentReplied'] = false;
 
-if (!is_object($serendipity['smarty'])) {
+if (!is_object($serendipity['smarty'] ?? null)) {
```

Comment:
Did this happen to you, that smarty wasn't initialized yet at this point?

Reply:
This happens to me at many places. Finally I decided to search for the pattern instead of fixing only crashing code. When you have better code here, please paste me.

Reply:
No, okay. As code it's fine, I was just surprised.